src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorEndPoint.class.php

```diff
@@ -2144,6 +2156,39 @@ protected function getFieldTypeIdBySystemName(string $systemName): ?string
         return $fieldTypeId;
     }
 
+    protected function getFieldsOfType(string $fieldTypeId): array
+    {
+        $databaseConnection = $this->getDatabaseConnection();
+
+        $fieldConfigQuery = 'SELECT 
+                         `cms_field_conf`.`name` AS fieldName,
+                         `cms_tbl_conf`.`name` AS tableName
+                    FROM `cms_field_conf` 
+               LEFT JOIN `cms_tbl_conf` ON `cms_tbl_conf`.`id` = `cms_field_conf`.`cms_tbl_conf_id` 
+                   WHERE `cms_field_conf`.`cms_field_type_id` = :fieldTypeId';
+
+        return $databaseConnection->fetchAll($fieldConfigQuery, ['fieldTypeId' => $fieldTypeId]);
+    }
+
+    protected function hasMultiTableRecordReferences(string $tableName, string $fieldName, string $deletedTableName, string $deletedRecordId = ''): bool
+    {
+        $databaseConnection = $this->getDatabaseConnection();
+
+        $relatedRecordsQuery = 'SELECT `id`
+                     FROM
```

Comment:
Write these lines in one line? Also above.

(SELECT
   FROM x
  WHERE y ... all aligned)

Reply:
reformatted the queries.

Reply:
Doesn't look like it? I meant SELECT x on one line.

Reply:
I don't get it. Maybe you can show me? The query is perfectly aligned in the code.

Reply:
done